README.md

```diff
@@ -1191,6 +1192,35 @@ f() // -> undefined
 
 You might expect `{}` instead of `undefined`. This is because the curly braces are part of the syntax of the arrow functions, so `f` will return undefined.
 
+## `arguments` and arrow functions
+
+Consider the example below:
+
+```js
+let f = function(){
+  return arguments;
+}
+f('a'); // -> ["a", callee: ƒ, ...]
```

Comment:
This expression doesn't return this for me. Look:

![image](https://user-images.githubusercontent.com/3459374/29640527-f8c07020-8867-11e7-9361-8b13e497ac69.png)